packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -389,7 +389,7 @@ export class AtomicResultList implements InitializableComponent {
   }
 
   public componentDidRender() {
-    if (this.resultListState.firstSearchExecuted) {
+    if (this.resultListState?.firstSearchExecuted) {
```

Comment:
Is it possible to add a test to avoid getting this bug again in the future?

Reply:
working on it